doc/adoc/user_guide.adoc

```diff
@@ -0,0 +1,102 @@
+= SUSE Distribution Migration System
+Marcus Schäfer; Jesús Velázquez
+
+:toc:
+:icons: font
+:numbered:
+
+:Authors: Marcus Schäfer and Jesús Bermúdez Velázquez
+:Publication_Date: TBD
+:Latest_Version: 0.5.15
+:Contributors: 
+:Repo: https://github.com/SUSE/suse-migration-services[suse-migration-services]
+
+ifdef::env-github[]
+//Admonitions
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+[%hardbreaks]
+Authors: {Authors}
+Contributors: {Contributors}
+Publication Date: {Publication_Date}
+Latest Version: {Latest_Version}
+Code available: {Repo}
+
+== Concept
+The Distribution Migration System provides an upgrade path for an installed SUSE Linux Enterprise system from one major version to another, for example, from SUSE Linux Enterprise Server 12 SP4 to SUSE Linux Enterprise Server 15. For a service pack upgrade from one Service Pack (SP) to another within a given major version release series the existing Zypper migration tool is the supported upgrade path. The distribution migration system provides the ability to upgrade across major distributions without the need to use the next major version installation media to perform the system upgrade.
+
+The upgrade to a new major versions requires the system to be offline during the upgrade to avoid system inconsistencies that may leave the system in a state that does not allow recovery. This behavior is implemented using a Live Migration Image.
+
+The distribution migration system provides the live image and a component that modifies the bootloader configuration such that on the next boot the system will boot into the upgrade image. Once booted into the upgrade live image the following chain of services will be executed:
+
+- detect the system to be migrated
+- mount the necessary file systems
+- setup the network to match the network configuration of the system to be migrated
+- prepare the package manager for the migration task
+- migrate the system
+- update the boot loader configuration
+- unmount all mounted file systems
+- reboot
```

Comment:
Sorry, I missed this (this will be my last comment, promised, please merge after fixing ;-)). Since this is a series of events that happen in order, we should  use an ordered list. The text also contains a few "migrations":

```
1. Detect the system to be migrated.
2. Mount the necessary file systems.
3. Setup the network to match the network configuration of the system to be upgraded.
4. Prepare the package manager for the upgrade task.
5. Upgrade the system.
6. Update the boot loader configuration.
7. Unmount all mounted file systems.
8. Reboot.
```